src/containers/Login/Login.js

```diff
@@ -65,16 +75,29 @@ export class UnconnectedLogin extends Component {
         }
       });
   }
+  
   handleChange(event) {
-    const { user } = this.state;
+    const { user, login } = this.state;
+    if ((validate('email', user.email) === false) & (user.password.length > 4)) {
```

Comment:
Hey @martinMutuma, nice work improving the UX, however, I can bypass your validation by entering blank spaces on the password field. Since you are using @misocho 's validation for email, you can also use the same for the password

```js
    if (
      (validate("email", user.email) === false) &
      ((validate("password", user.password) === false) && user.password.length > 4)
    )
```

Reply:
We should not validate password at login because it presents a security vulnerability since we are telling a hacker how our password looks like

Reply:
I agree with @martinMutuma. Validating blank spaces would be a security vulnerability.